src/app/components/HomePage.tsx

```diff
@@ -38,30 +49,95 @@ const namespaceFindYour: Record<Namespace, string> = {
 };
 
 const HomePage = () => {
-  const namespace = useNS();
+  const prefersReducedMotion = useReducedMotion();
 
   const [selectedNamespace, setSelectedNamespace] = useState(
-    namespace || Namespace.uniprotkb
+    Namespace.uniprotkb
   );
 
+  const text = namespaceFindYour[selectedNamespace];
+  const [displayed, setDisplayed] = useState(text);
+
+  const interval = useRef<number>();
+
+  const firstRender = useRef(true);
+  useEffect(() => {
+    if (firstRender.current) {
+      firstRender.current = false;
+      return;
+    }
+    setDisplayed('');
+  }, [selectedNamespace]);
+
+  useEffect(() => {
+    if (prefersReducedMotion) {
+      setDisplayed(text);
+      return;
+    }
+    window.clearInterval(interval.current);
+    interval.current = window.setInterval(() => {
+      setDisplayed((displayed) => text.substr(0, displayed.length + 1));
+    }, 100);
+  }, [prefersReducedMotion, text]);
+
+  if (displayed === text) {
+    window.clearInterval(interval.current);
+  }
+
   return (
     <>
       <main>
         <ErrorBoundary>
           <HeroHeader
-            title={`Find your ${namespaceFindYour[selectedNamespace]}`}
+            className="home-page__header"
+            title={
+              <>
+                {'Find your '}
+                <button
+                  type="button"
+                  className="letter-group"
+                  tabIndex={-1}
+                  onClick={useCallback(() => {
+                    const dropdown: HTMLButtonElement | null = document.querySelector(
+                      'form.main-search button.dropdown'
+                    );
+                    dropdown?.focus();
+                    dropdown?.click();
+                  }, [])}
+                >
+                  {prefersReducedMotion ? (
+                    displayed
+                  ) : (
+                    <>
+                      {displayed.split('').map((letter, index) => (
+                        // eslint-disable-next-line react/no-array-index-key
+                        <span key={index} className="letter">
+                          {letter}
+                        </span>
+                      ))}
+
+                      <span className="cursor">|</span>
```

Comment:
Wondering if the cursor should disappear after the word has been written?

Reply:
Not sure if you've noticed but it's only flashing if the user's focus is within the hero header, or if their mouse is. The logic being to grab the attention on alternative actions (because clicking the text would expose the dropdown) is case the user is stuck looking at the header and wondering what they can do. If they're doing anything with the buttons at the top or with the rest of the page, I'm assuming they found something to do, so I don't grab the attention.
What I could do:
 - Change this to flashing only on hover
 - Just flash a couple of times then disappear

Reply:
Wondering if flashing on hover could mean "You can edit me" so I would favour the second option?

Reply:
Well, that the point. "Oh can I edit that? Let's try." clicks on it, it opens the dropdown "Oh, so _that_'s how I can 'edit' this"

Reply:
That's not editing the text in place though (which I would think is the expected behaviour when you see a blinking cursor).

Reply:
Well, that can be removed altogether anyway. This PR started as an experiment, I'm not attached to the cursor.